.github/workflows/_test_te.yaml

```diff
@@ -0,0 +1,120 @@
+name: ~test TransformerEngine
+
+on:
+  workflow_call:
+    inputs:
+      JAX_IMAGE:
+        type: string
+        description: 'JAX image'
+        required: true
+      ARTIFACT_PREFIX:
+        type: string
+        description: 'Name of the artifact zip file'
+        required: false
+        default: 'te-multi-gpu'
+
+jobs:
+  te-multi-gpu:
+    uses: ./.github/workflows/_test_slurm_pyxis.yaml
+    strategy:
+      matrix:
+        N_GPU: [8]
+      fail-fast: false
+    secrets:
+      SSH_PRIVATE_KEY: ${{ secrets.SSH_PRIVATE_KEY }}
+      SLURM_LOGIN_USER: ${{ secrets.CLUSTER_LOGIN_USER }}
+      CONTAINER_REGISTRY_TOKEN: ${{ secrets.github_token }}
+    with:
+      NAME: ${{ inputs.ARTIFACT_PREFIX }}-${{ matrix.N_GPU }}
+      SLURM_LOGIN_HOSTNAME: ${{ vars.HOSTNAME_SLURM_LOGIN }}
+      OUTPUT_BASEDIR: /nfs/cluster
+      OUTPUT_MOUNTPOINT: /output
+      NODES: 1
+      GPUS_PER_NODE: ${{ matrix.N_GPU }}
+      NTASKS: 1
+      NTASKS_PER_NODE: 1
+      TIME_LIMIT: '00:30:00'
+      EXTRA_EXPORTS: 'VOCAB_PATH=gs://t5-data/vocabs/cc_all.32000.100extra/sentencepiece.model'
+      IMAGE: ${{ inputs.JAX_IMAGE }}
+      SRUN_PREAMBLE: |
+        nvidia-smi
+        pip install \
+          pytest \
+          pytest-reportlog \
+          cuda-python \
+          -r ${SRC_PATH_TRANSFORMER_ENGINE}/examples/jax/encoder/requirements.txt
+      SRUN_SCRIPT: |
+        set -ex
+        cd ${SRC_PATH_TRANSFORMER_ENGINE}/examples/jax/encoder
+        pytest -vvv --report-log=/output/pytest-report.jsonl \
+          test_single_gpu_encoder.py \
+          test_multigpu_encoder.py \
+          test_model_parallel_encoder.py
```

Comment:
These are part of https://github.com/NVIDIA/TransformerEngine/blob/4f33ece48b542ac29b5a483ffafc2245cb6a7334/qa/L0_jax_distributed_unittest/test.sh#L25-L26